src/API.jl

```diff
@@ -894,71 +894,88 @@ end
 
 precompile() = precompile(Context())
 function precompile(ctx::Context)
+    
+    function is_stale(paths, sourcepath)
+        toml_c = Base.TOMLCache()
+        for path_to_try in paths::Vector{String}
+            staledeps = Base.stale_cachefile(sourcepath, path_to_try, toml_c)
+            staledeps ? continue : return false
```

Comment:
`stale_cachefile` can return non-boolean: https://github.com/JuliaLang/julia/blob/c0d06d3219751d28237cb32d9d6f7e2ef45916cd/base/loading.jl#L1428-L1429

```suggestion
            staledeps === true ? continue : return false
```

Reply:
ah yes, thanks

Reply:
Sorry for the bad review, I just thought `x === true` looked weird.